content/blog/how-tikv-accesses-data.md

```diff
@@ -0,0 +1,263 @@
+---
+title: How TiKV reads and writes
+date: 2019-12-06
+author: Tang Liu
+---
+
+This article introduces in detail how TiKV handles read and write operations. After reading this document, you will learn how TiKV, as a distributed database, stores the data contained in a write request and how it retrieves the corresponding data with consistency guaranteed.
+
+## Before you read it
+
+Before we begin, we need to introduce some essential concepts of TiKV to help you fully understand the process. If you are already familiar with these, you can skip to the next section.
+
+### Raft
+
+{{< figure
+    src="/img/blog/how-tikv-accesses-data/raft-process-in-tikv.png"
+    caption="Raft Process in TiKV"
+    number="" >}}
+
+TiKV uses the Raft consensus algorithm to ensure data safety and consistency. By default, three replicas are supported, and these three replicas form a Raft Group.
+
+When the client needs to write some data, it sends the request to the Raft Leader. This process is called **Propose** in TiKV. The Leader encodes the operation into an entry and writes it into its own **Raft Log**. This is called **Append**.
```

Comment:
```suggestion
When the client needs to write some data, it sends the request to the Raft Leader. This process is called **Propose** in TiKV. The Leader encodes the operation into an entry and writes it into its own Raft Log. This is called **Append**.
```

The boldness on this seems a bit inconsistent, all the other bolds are commands or events, while this is a module or object.